### ISSUE TITLE
Install of zipline using anaconda results in Performance Warnings for every command

### ISSUE THREAD

**Issue by blair-anson** [quantopian/zipline#1658]
Dear Zipline Maintainers,

Before I tell you about my issue, let me describe my environment:

# Environment

* Operating System: `Linux  4.4.0-59-generic #80-Ubuntu SMP Fri Jan 6 17:47:47 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux`
* Python Version: `Python 3.4.5 :: Continuum Analytics, Inc.`
* How did you install Zipline: `conda`
* Python packages: 
$ conda list
```
# packages in environment at /home/b/anaconda3/envs/py3.4:
#
alembic                   0.7.7                    py34_0    Quantopian
bcolz                     0.12.1              np110py34_0  
bottleneck                1.0.0               np110py34_1  
cachetools                1.1.6                    py34_0    Quantopian
cairo                     1.14.8                        0  
click                     6.7                      py34_0  
contextlib2               0.5.4                    py34_0  
cycler                    0.10.0                   py34_0  
cyordereddict             0.2.2                    py34_0    Quantopian
cython                    0.25.2                   py34_0  
decorator                 4.0.11                   py34_0  
empyrical                 0.2.1                    py34_0    Quantopian
fontconfig                2.12.1                        2  
freetype                  2.5.5                         2  
glib                      2.50.2                        1  
harfbuzz                  0.9.39                        2  
intervaltree              2.1.0                    py34_0    Quantopian
ipython                   5.1.0                    py34_0  
ipython_genutils          0.1.0                    py34_0  
libffi                    3.2.1                         1  
libgfortran               3.0.0                         1  
libiconv                  1.14                          0  
libpng                    1.6.27                        0  
libxml2                   2.9.4                         0  
logbook                   0.12.5                   py34_0    Quantopian
mako                      1.0.6                    py34_0  
markupsafe                0.23                     py34_2  
matplotlib                1.5.1               np110py34_0  
mkl                       11.3.3                        0  
multipledispatch          0.4.9                    py34_0  
networkx                  1.11                     py34_0  
numexpr                   2.6.0               np110py34_0  
numpy                     1.10.4                   py34_2  
openssl                   1.0.2j                        0  
pandas                    0.17.1              np110py34_0  
pandas-datareader         0.2.1                    py34_0  
pango                     1.40.3                        1  
path.py                   10.0                     py34_0  
patsy                     0.4.1                    py34_0  
pcre                      8.39                          1  
pexpect                   4.2.1                    py34_0  
pickleshare               0.7.4                    py34_0  
pip                       9.0.1                    py34_1  
pixman                    0.34.0                        0  
prompt_toolkit            1.0.9                    py34_0  
ptyprocess                0.5.1                    py34_0  
pygments                  2.1.3                    py34_0  
pyparsing                 2.1.4                    py34_0  
pyqt                      4.11.4                   py34_4  
python                    3.4.5                         0  
python-dateutil           2.6.0                    py34_0  
pytz                      2016.10                  py34_0  
qt                        4.8.7                         4  
readline                  6.2                           2  
requests                  2.12.4                   py34_0  
requests-file             1.4.1                    py34_0  
scipy                     0.17.1              np110py34_1  
seaborn                   0.7.1                    py34_0  
setuptools                27.2.0                   py34_0  
simplegeneric             0.8.1                    py34_1  
sip                       4.18                     py34_0  
six                       1.10.0                   py34_0  
sortedcontainers          1.4.4                    py34_0    Quantopian
sqlalchemy                1.1.5                    py34_0  
sqlite                    3.13.0                        0  
statsmodels               0.6.1               np110py34_1  
tk                        8.5.18                        0  
toolz                     0.8.2                    py34_0  
traitlets                 4.3.1                    py34_0  
wcwidth                   0.1.7                    py34_0  
wheel                     0.29.0                   py34_0  
xz                        5.2.2                         1  
zipline                   1.0.2               np110py34_0    Quantopian
zlib                      1.2.8                         3 
```

Now that you know a little about me, let me tell you about the issue I am
having:

# Description of Issue

* What did you expect to happen?
zipline to run without warnings, even when doing `zipline --help`

* What happened instead?
pandas performance warnings are shown

Here is how you can reproduce this issue on your machine:

## Reproduction Steps
1. Install zipline using anaconda in a python 3.4 environment (as per the Ziping install instructions)
2. run any zipline command...

```
$ zipline --help
/home/b/anaconda3/envs/py3.4/lib/python3.4/site-packages/pandas/tseries/index.py:755: PerformanceWarning: Non-vectorized DateOffset being applied to Series or DatetimeIndex
  PerformanceWarning)
/home/b/anaconda3/envs/py3.4/lib/python3.4/site-packages/pandas/tseries/index.py:755: PerformanceWarning: Non-vectorized DateOffset being applied to Series or DatetimeIndex
  PerformanceWarning)
/home/b/anaconda3/envs/py3.4/lib/python3.4/site-packages/pandas/tseries/index.py:755: PerformanceWarning: Non-vectorized DateOffset being applied to Series or DatetimeIndex
  PerformanceWarning)
Usage: zipline [OPTIONS] COMMAND [ARGS]...

  Top level zipline entry point.

Options:
  -e, --extension TEXT            File or module path to a zipline extension
                                  to load.
  --strict-extensions / --non-strict-extensions
                                  If --strict-extensions is passed then
                                  zipline will not run if it cannot load all
                                  of the specified extensions. If this is not
                                  passed or --non-strict-extensions is passed
                                  then the failure will be logged but
                                  execution will continue.
  --default-extension / --no-default-extension
                                  Don't load the default zipline extension.py
                                  file in $ZIPLINE_HOME.
  --help                          Show this message and exit.

Commands:
  bundles  List all of the available data bundles.
  clean    Clean up data downloaded with the ingest...
  ingest   Ingest the data for the given bundle.
  run      Run a backtest for the given algorithm.
```

## What steps have you taken to resolve this already?
Tried downgrading numpy as per #1485, but could not as zipline conflict states `- zipline -> numpy 1.10*`

# Anything else?
I suspect this is a regression similar to #1485

**Comment by sabr-p** [comment]
(zipline) ➜  notebooks $ conda info --envs
# conda environments:
#
zipline               *  /home/mehere/anaconda3/envs/zipline
root                     /home/mehere/anaconda3
(zipline) ➜  notebooks $ conda --version
conda 4.2.9
(zipline) ➜  notebooks $ python --version
Python 3.4.5 :: Continuum Analytics, Inc.

confirm the issue here, zipline installed in a fresh conda installation

**Comment by Georgantas** [comment]
Downgrade pandas to 0.17.0. That should fix the warnings.

**Comment by blair-anson** [comment]
Yep, that did the trick.
Conda command to downgrade was... `conda install pandas=0.17.0`

**Comment by FreddieV4** [comment]
We've just released zipline `1.1.0` which support pandas 0.18
